packages/example/src/data/nav-items.yaml

```diff
@@ -48,6 +48,7 @@
       path: /guides/v3-migration
     - title: v4.0 Migration
       path: /guides/v4-migration
+  hasDivider: true
```

Comment:
Was this added on purpose?

Reply:
I added it for testing purpose only. I have now removed it.